benchmark/aloha_cloth/scene.xml

```diff
@@ -0,0 +1,345 @@
+<mujoco model="aloha with cloth">
+  <compiler autolimits="true" boundmass="0.01" boundinertia="9.9999999999999995e-07" angle="radian" meshdir="../aloha_pot"/>
+  <option impratio="10" cone="elliptic" timestep="0.0001" solver="CG" tolerance="1e-6" jacobian="sparse"/>
```

Comment:
do we want/need `elliptic` friction cone? if not, we probably don't need `impratio=10`